TestFoundation/TestFileManager.swift

```diff
@@ -492,10 +497,17 @@ class TestFileManager : XCTestCase {
         let fm = FileManager.default
         let basePath = NSTemporaryDirectory() + "testdir\(NSUUID().uuidString)/"
         let subDirs1 = basePath + "subdir1/subdir2/.hiddenDir/subdir3/"
-        let subDirs2 = basePath + "subdir1/subdir2/subdir4.app/subdir5./.subdir6.ext/subdir7.ext./"
         let itemPath1 = basePath + "itemFile1"
+#if os(Windows)
+        // Filenames ending with '.' are not valid on Windows, so don't bother testing them
```

Comment:
@millenomi - what do you think about this?

Reply:
That's fine.